### PR TITLE
fix(@desktop/app-search): add color hash to chat icons

### PR DESCRIPTION
**Pull request by osmaczko**
closes: #5459
needs: ~~https://github.com/status-im/StatusQ/pull/701~~

https://user-images.githubusercontent.com/33099791/171131163-fd4120e9-d227-4075-83a2-0d55bf95e68e.mp4

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]
Looks straight forward

File: src/app/modules/main/app_search/location_menu_sub_item.nim [hunk 1]
```diff
@@ -1,14 +1,26 @@
-import json, strformat
+import json, strformat, sequtils, sugar
 import base_item
+import ../../shared_models/[color_hash_item, color_hash_model]
 
 export base_item
 
+type
+  ColorHashSegment* = tuple[len, colorIdx: int]
```
Comment (endulab) [inline src/app/modules/main/app_search/location_menu_sub_item.nim:8]: You define this type here and in result_item. Maybe you can move it to color_hash_model ?

**Review by endulab — APPROVED** [review]

File: src/app/modules/main/app_search/location_menu_sub_item.nim (hunk 1 above)
Comment (osmaczko) [reply src/app/modules/main/app_search/location_menu_sub_item.nim]: Thanks, very good suggestion, applied :+1:

**Comment by osmaczko** [comment]
@endulab since QAs are busy could you please also confirm (test) it's working?

**Comment by endulab** [comment]
@osmaczko works correctly. You can merge.
image

**Pull request merged**